examples/dataflow-streaming-benchmark/README.md

```diff
@@ -39,6 +39,11 @@ instruct the data generator of what type of fake data to create in each field. S
 json-data-generator [docs](https://github.com/vincentrussell/json-data-generator) for more
 information on the faker functions.
 
+#### Message Attributes
+In case schema contains fields matching (case insensitive) names eventId, eventTimestamp then such fields
+will be added to attributes.  Attribute fields can be helpful in various scenarios like deduping messages, 
+inspecting message timestamps etc
```

Comment:
Cleaning up grammar... I would update it to something like the follow:

```
If the message schema contains fields matching (case-insensitive) the following names then such fields
will be added to the output Pub/Sub message attributes:
eventId, eventTimestamp

Attribute fields can be helpful in various scenarios like deduping messages, inspecting message timestamps, etc.
```

Reply:
Done